AzureCalling/Controllers/InviteViewController.swift

```diff
@@ -0,0 +1,105 @@
+//  Licensed under the MIT License.
+//
+import UIKit
+import FluentUI
+
+class InviteViewController: UIViewController {
+    // MARK: Properties
+    private var iconImageView: UIImageView!
+    private var titleLabel: FluentUI.Label!
+    private var subtitleLabel: FluentUI.Label!
+    private var shareButton: FluentUI.Button!
+    private var continueButton: FluentUI.Button!
+
+    // MARK: ViewController Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+
+    // MARK: UI layout
+    private func setupUI() {
+        view.backgroundColor = ThemeColor.lightSurfacesPrimary
+        title = "Invite another device ?"
+        setupImageViews()
+        setupLabels()
+        setupButtons()
+        layoutContinueButton()
+        layoutMainContainer()
+    }
+
+    private func setupImageViews() {
+        let imageRect = CGRect(x: 0, y: 0, width: 27, height: 23)
+        iconImageView = UIImageView(frame: imageRect)
+        iconImageView.contentMode = .scaleAspectFit
+        iconImageView.tintColor = FluentUI.Colors.iconSecondary
+        iconImageView.image = UIImage(named: "ic_fluent_phone_desktop_28_regular")
+    }
+
+    private func setupLabels() {
+        titleLabel = FluentUI.Label.createWith(style: .title1,
+                                               colorStyle: .regular,
+                                               value: "Invite another device?")
+        subtitleLabel = FluentUI.Label.createWith(style: .body,
+                                                  colorStyle: .regular,
+                                                  value: "Use the group call ID to join this call on another device.")
+        subtitleLabel.numberOfLines = 0
+        subtitleLabel.textAlignment = .center
+    }
+
+    private func setupButtons() {
+        shareButton = FluentUI.Button.createWith(
+            style: .secondaryOutline,
+            title: "Share group call ID",
+            action: { [weak self] _ in self?.onShareButtonTapped() }
+        )
+        shareButton.image = UIImage(named: "ic_fluent_share_screen_stop_20_regular")
+
+        continueButton = FluentUI.Button.createWith(
+            style: .primaryFilled,
+            title: "Continue",
+            action: { [weak self] _ in self?.onContinueButtonTapped() }
+        )
+    }
+
+    private func layoutContinueButton() {
+        view.addSubview(continueButton)
+        continueButton.fixHeightTo(height: 52)
+        continueButton.expandHorizontallyInSuperView(withEqualMargin: 16)
+        continueButton.pinToBottom(withMargin: 16)
+    }
+
+    private func layoutMainContainer() {
+        let mainContainer = UIView()
+        view.addSubview(mainContainer)
+        mainContainer.expandHorizontallyInSuperView(withEqualMargin: 16)
+        mainContainer.pinToTop(withMargin: 0)
+        mainContainer.bottomAnchor.constraint(equalTo: continueButton.topAnchor, constant: -12).isActive = true
```

Comment:
A minor thing, but IMHO having positive constants is easier to understand, which can be done by reversing the anchors